src/Analyzers/VisualBasic/Tests/NamespaceMatchFolder/VisualBasicNamespaceMatchFolderTests.vb

```diff
@@ -0,0 +1,128 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.IO
+Imports VerifyVB = Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions.VisualBasicCodeFixVerifier(Of
+    Microsoft.CodeAnalysis.VisualBasic.Analyzers.NamespaceMatchFolder.VisualBasicNamespaceMatchFolderDiagnosticAnalyzer,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider)
```

Comment:
I'm a little confused about what we are testing here. Is this a codefix verifier that verifies a dummy provider?

Reply:
This seems to be the standard way to test an analyzers without a codefix provider. The EmptyCodeFixProvider doesn't add any fixes

Reply:
i'm confused by this.  don't you have a fix provider?

Reply:
Removed VB impl